### ISSUE TITLE
Duplicate export 'default'

### ISSUE THREAD

**Issue by DaveWM** [rollup/rollup-plugin-commonjs#158]
It seems that when a module sets both `module.exports` and `module.exports.default`, rollup throws the above error. This is needed for node to be able to import commonjs modules using es6, see [this discussion of the issue.](https://github.com/esnext/es6-module-transpiler/issues/85)

I've put together a gist which outlines the issue [here](https://gist.github.com/DaveWM/73ed0a6284510ecb86b2ee0117564359). When you run `rollup -c`, you get the following error:
```
🚨   Duplicate export 'default'
x.js (8:22)
6: export default x_1;
7: export { x_1 as __moduleExports };
8: export { default_1 as default };
                         ^
9: x_1.default = default_1;
```

Anyone know what's happening here?

**Comment by mortargrind** [comment]
I think this might be related to #110

**Comment by Rich-Harris** [comment]
Thanks for the repro and sorry for the long delay — this is fixed in 7.0.2
